otri/filtering/filters/split_filter.py

```diff
@@ -89,47 +118,74 @@ class SwitchFilter(Filter):
     ordering is not.
     '''
 
-    def __init__(self, source_stream: Stream, key: Any, cases: Set, ignore_none: bool = True):
+    def __init__(self, input: str, cases_output : Sequence[str], default_output : str, key: Any, cases: Set, none_keys_output: str = None):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'input'](https://app.codacy.com/gh/OTRI-Unipd/OTRI/pullRequest?prid=5649486)